src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

```diff
@@ -231,7 +233,39 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
         ''' </summary>
         Private Sub NeutralLanguageComboBox_DropDown(sender As Object, e As EventArgs) Handles NeutralLanguageComboBox.DropDown
             PopulateNeutralLanguageComboBox(NeutralLanguageComboBox)
-            Common.SetComboBoxDropdownWidth(NeutralLanguageComboBox)
+            SetComboBoxDropdownWidth(NeutralLanguageComboBox)
+        End Sub
+
+        Private Sub LicenseBrowseButton_Click(sender As Object, e As EventArgs) Handles LicenseBrowseButton.Click
+            Dim sInitialDirectory As String = Nothing
```

Comment:
We don't typically use prefixes for simple data types like `String`.

Reply:
Good point, fixed.

This code was borrowed for testing purposes from ApplicationPropPage.vb, which seems to not have changed in a meaningful way since 2003. I am going to clean up the whole function and I should probably make a separate PR for cleaning up the other PropPages as well.